scripts/coverage

```diff
@@ -8,4 +8,4 @@ export SOURCE_FILES="broadcaster tests"
 
 set -x
 
-${PREFIX}coverage report --show-missing --skip-covered --fail-under=78
+${PREFIX}coverage report
```

Comment:
Can we also remove the `SOURCE_FILES` from this file, please?

Just noticed, sorry.

Reply:
No worries :-)